src/openms/include/OpenMS/METADATA/SpectrumSettings.h

```diff
@@ -142,6 +151,7 @@ namespace OpenMS
 protected:
 
     SpectrumType type_;
+    IMFormat im_type_;
```

Comment:
SpectrumSettings is already a whopping 656 bytes per spectrum, i.e. worth about 54 peaks. We should do something about that, but these 4 bytes won't make a difference.